### ISSUE TITLE
RIAW RC 1.5.0-rc.3 is missing the a11y view

### ISSUE THREAD

**Issue by zepumph** [phetsims/perennial#89]
The previous version has the a11y view built in, https://phet-dev.colorado.edu/html/resistance-in-a-wire/1.5.0-rc.1/phet and so does a more recent 1.6.0-dev.1.

But 
https://phet-dev.colorado.edu/html/resistance-in-a-wire/1.5.0-rc.3/phet/ does not include it.

When I built the sim locally beforehand it was built. @jonathanolson and I thought it was a build-server issue. @mattpen what do you think? I just build about 15 minutes ago.

For now, I'm going to just copy the locally built one up to this rc. It is a bit of a hack, but both builds were from the same shas, so I'm not too worried.

**Comment by mattpen** [comment]
@zepumph - What is the build command I can use to recreate this?

**Comment by zepumph** [comment]
`grunt generate-a11y-view-html` will put an a11y view into your sim dir you run it from. On build we use `getA11yViewHTMLFromTemplate` in `buildRunnable` to get a string we can write to the build dir. I hope that is helpful.

**Comment by mattpen** [comment]
This is related to the translated sim filter for rc deploys and the dev copy of production deploys.  I updtaed the filter to allow files with "_a11y" in the name and deployed this to production.  @zepumph is going to test later today.

**Comment by zepumph** [comment]
Looks great thanks @mattpen 

![image](https://user-images.githubusercontent.com/6856943/40021063-4e5320fa-5770-11e8-9471-f66809fd7b52.png)

**Comment by zepumph** [comment]
@mattpen did this ever get deployed to the build server. This a11y view isn't showing up for us right now.

**Comment by mattpen** [comment]
@zepumph - It is showing up for me and looks like it was created at build time, can you check again?

**Comment by zepumph** [comment]
Yeah I don't think this is the stem of an issue I'm investigating, sorry to bother.
